linux/opt/sage/bin/apache_conf_rewrite.sh

```diff
@@ -5,6 +5,7 @@
 # notebook gets stripped out before the traffic is received by the notebook web server.
 # We did it this way because the Apache module we use for rewrite did not support regular expressions.
 
-EC2_INSTANCE_ID=$(/usr/bin/curl -s http://169.254.169.254/latest/meta-data/instance-id)
+TOKEN=$(curl -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 21600")
```

Comment:
you've removed all the paths to these executables.  i believe this is running with the root user.  have you tested with the root user to make sure it works without the paths?

Reply:
yes, I've confirmed that both /usr/bin and /usr/local/bin are in root's $PATH